MCprep_addon/spawner/item.py

```diff
@@ -19,8 +19,13 @@
 import os
 
 import bpy
+from bpy.types import (
+  Context, Object
+)
```

Comment:
Just import `Context` as `Object` on its own gets confusing (for a reason I'll get to later)
```suggestion
from bpy.types import Context
```